spec/data/approved_tags.json

```diff
@@ -0,0 +1,150 @@
+{
+    "approved_http_tags": [
+        "2FAServer",
+        "Accounting",
+        "ADC",
+        "Admin Panel",
+        "Administrative",
+        "Analytics",
+        "API",
+        "API Gateway",
+        "APIServer",
+        "APM",
+        "Appliance",
+        "ApplicationServer",
+        "ASGIServer",
+        "AuthenticationServer",
+        "AuthorizationServer",
+        "B2B",
+        "BaaS",
+        "Cache",
+        "Caching Proxy",
+        "CAS",
+        "CASB",
+        "CDN",
+        "CI/CD",
+        "Cloud",
+        "CMS",
+        "Collaboration",
+        "Communication",
+        "Compliance",
+        "COTS",
+        "CRM",
+        "Customer Service",
+        "DAM",
+        "Database",
+        "Development",
+        "DMS",
+        "DNSServer",
+        "Documentation",
+        "Domain Management",
+        "DRM",
+        "Drupal Plugin",
+        "E-Commerce",
+        "E-Learning",
+        "EbookServer",
+        "Editor",
+        "Education",
+        "Email",
+        "Email Encryption",
+        "Embedded",
+        "Engineering",
+        "Entertainment",
+        "Extension",
+        "Fax",
+        "Finance",
+        "Firewall",
+        "Firmware",
+        "Forum",
+        "Framework",
+        "Frontend",
+        "FTP",
+        "FTPServer",
+        "Hardware",
+        "HCM",
+        "Health",
+        "Hospitality",
+        "Hosting",
+        "HR",
+        "HTTP Compression",
+        "HTTP Router",
+        "Hypervisor",
+        "IaaS",
+        "IAM",
+        "IndexServer",
+        "Intranet",
+        "IoT",
+        "Issue Tracker",
+        "JavaScript",
```

Comment:
*Javascript

Reply:
![image](https://user-images.githubusercontent.com/10855429/117470459-bc83db80-af4e-11eb-834f-4ad802239e75.png)

Reply:
dang. Altright, you win this one